iceoryx_hoofs/test/moduletests/test_cxx_string.cpp

```diff
@@ -2138,6 +2191,44 @@ TYPED_TEST(stringTyped_test, ConcatenateStringLiteralAndStringWithOperatorPlusWo
     EXPECT_THAT(testString.c_str(), StrEq("AdmTasse"));
 }
 
+TYPED_TEST(stringTyped_test, ConcatenateEmptyStringAndCharWithOperatorPlusWorks)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "09efee5c-985f-4852-aa3c-5c67fc61071b");
+    using MyString = typename TestFixture::stringType;
+    constexpr auto STRINGCAP = MyString::capacity();
+    string<2U * STRINGCAP> testString = this->testSubject + 'M';
+    EXPECT_THAT(testString.capacity(), Eq(2U * STRINGCAP));
```

Comment:
We should not test that here. This is checked in tests of the `capacity` function. We explicitly created a string with this capacity.